### DISCUSSION TITLE
Feature: Individual Column search in datatable

### DISCUSSION THREAD

**Discussion by shahpankaj981** [bootstrap-vue/bootstrap-vue#5881] [Q&A] [answered]
Does this package have individual column search in datatables like in the image at [https://pasteboard.co/JuWDrGt.png](https://pasteboard.co/JuWDrGt.png)

It can also be seen at [https://datatables.net/examples/api/multi_filter.html](url)

**Comment by Hiws** [comment, marked as answer]
This is not something that is supported currently.

You would have to do a custom implementation.
